src/full/Agda/Syntax/Internal/SanityCheck.hs

```diff
@@ -34,24 +34,34 @@ sanityCheckSubst gamma rho delta = go gamma rho delta
   where
     go gamma rho delta =
       case rho of
-        IdS      -> unless (size gamma == size delta) $ err $ "idS:" <+> hang (pretty gamma <+> "/=") 2 (pretty delta)
-        EmptyS _ -> unless (size delta == 0) $ err $ "emptyS:" <+> pretty delta <+> "is not empty"
+
+        IdS      -> do
+          unless (natSize gamma == natSize delta) $ err $
```

Comment:
This test is expected to succeed, right? In that case I don't see the point of this change. (This module seems to be unused, so it could perhaps be removed.)

Reply:
Looking at the history, this module was added in 2016 by @UlfNorell for debugging purposes and then mechanically updated in refactorings.
I don't find even commented out references to the functions defined in this module, so maybe it has fulfilled its service and can be removed.

Reply:
I reverted my changes to this module.  Removing it is not in scope of this PR.